src/request_handler_sql.cpp

```diff
@@ -0,0 +1,143 @@
+//
+// request_handler_sql.cpp
+// ~~~~~~~~~~~~~~~~~~~
+//
+
+#include "request_handler_sql.hpp"
+#include <string>
+#include <fstream>
+#include <sstream>
+#include <string>
+#include <boost/lexical_cast.hpp>
+#include "mime_types.hpp"
+#include <iostream>
+#include <mysql_connection.h>
+#include <cppconn/driver.h>
+#include "response.hpp"
+#include "request.hpp"
+#include "request_handler.hpp"
+#include "sql_engine.hpp"
+
+namespace http {
+namespace server {
+
+RequestHandler::Status SqlHandler::Init(const std::string& uri_prefix, const NginxConfig& config) {
+	// Get root directory from config.
+	  std::vector<std::shared_ptr<NginxConfigStatement>> statements = config.statements_;
```

Comment:
spacing of this file is a little off -- I think all of these statements are being indented by an extra few spaces?